package-lock.json

```diff
@@ -672,9 +828,9 @@
       }
     },
     "node_modules/@typescript-eslint/eslint-plugin/node_modules/semver": {
-      "version": "7.3.7",
-      "resolved": "https://registry.npmjs.org/semver/-/semver-7.3.7.tgz",
-      "integrity": "sha512-QlYTucUYOews+WeEujDoEGziz4K6c47V/Bd+LjSSYcA94p+DmINdf7ncaUinThfvZyu13lN9OY1XDxt8C0Tw0g==",
+      "version": "7.6.0",
```

Comment:
On line 2238 in this file, we're still locked to semver v3.6.0.

If you run `npm i semver@6.3.1`, we should be at v3.6.1 or higher everywhere. Can you give this a try?

Reply:
Thanks for catching, I'll go through and check the rest more thoroughly 😞

Reply:
`npm why semver` helps. I only see 6.3.1 and 7.6.0 now, which looks good to me.